gateway/src/apicast/policy/url_rewriting/schema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "URL rewriting policy configuration",
+  "type": "object",
+  "properties": {
+    "commands": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "op": {
+            "type": "string",
+            "enum": ["sub", "gsub"]
+          },
+          "regex": {
+            "type": "string"
+          },
+          "replace": {
+            "type": "string"
+          },
+          "options": {
+            "type": "string"
+          },
+          "break": {
```

Comment:
We are considering also other mutually exclusive values like: `last`, `redirect`, `persistent` right?

IMO might be nicer to model it in a way they are truly mutually exclusive by being a value of one key: `"type": "break"`. 

But since we are not releasing this yet, It could wait until we have some more options.